schemas.sql

```diff
@@ -0,0 +1,112 @@
+
+-- keep track of all users (id, tag, name)
+
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    user_id VARCHAR NOT NULL,
```

Comment:
Is this the Discord user id? If so we should label it `discord_user_id` or `user_id_discord` or something.